config-templates/redhat/ocs-remote/4.6/README.md

```diff
@@ -0,0 +1,317 @@
+# Red Hat OpenShift Container Storage - Remote Storage
+
+Red Hat OpenShift Container Storage is a software-defined storage that is optimised for container environments. It runs as an operator on OpenShift Container Platform to provide highly integrated and simplified persistent storage management for containers.
+
+The user has to provide the input values to the custom resource OcsCluster while creating the satellite configuration to deploy OCS
+
+## Prerequisites
+In order to deploy OCS, the following prerequisites are required.
+- Remote storage available in block mode
+- Your cluster must have a minimum of 3 nodes that each have a minimum 16CPUs and 64GB RAM.
+- Your cluster should be compatible with the OCS version that you're trying to install.
+- The storage class you use for the `mon-storage-class` and `osd-storage-class` parameters should have `VolumeBindingMode` set to `WaitForFirstConsumer`
+- You must provision an instance of IBM Cloud Object Storage and provide your COS HMAC credentials, the regional public endpoint, and the IBM COS location when you create your storage configuration.
+
+### Creating the IBM COS service instance
+
+Run the following commands to create a COS instance and create a set of HMAC credentials.
+
+1. Create an IBM COS Service Instance.
+    ```
+    $ ibmcloud resource service-instance-create noobaa-stor cloud-object-storage standard global
+    ```
+
+2. Create HMAC credentials. Make a note of your credentials.
+    ```
+    $ ibmcloud resource service-key-create cos-cred-rw Writer --instance-name noobaa-stor --parameters '{ "HMAC": true}'
+    ```
+
+3. Get the regional public endpoint and the location of your IBM COS instance.
+    You can get the endpoint details from the list of [IBM COS endpoints](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-endpoints) and the location from [IBM COS location constraint details](https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-classes).
+
+## Red hat Openshift Container Storage - Remote Storage: Parameter reference
+
+**Description of the template parameters :**
+
+| Parameter | Required? | Description | Default value if not provided | Datatype |
+| --- | --- | --- | --- | --- |
+| `ocs-cluster-name` | Required | Enter the name of your OcsCluster custom resource. | N/A | string |
+| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
```

Comment:
```suggestion
| `mon-storage-class` | Required | Enter the storage class name that you want to use for the mon pods | N/A | string |
```

^ for the storage class - if the cluster is multizone does it need to be a metro class with waitforfirstconsumer binding mode? - i see the command below uses a custom class